terraform/050-pw-manager/task-definition-api.json

```diff
@@ -70,14 +70,22 @@
         "value": "${help_center_url}"
       },
       {
-        "name": "ID_BROKER_ACCESS_TOKEN",
+        "name": "ID_BROKER_accessToken",
         "value": "${id_broker_access_token}"
       },
       {
-        "name": "ID_BROKER_BASE_URI",
+        "name": "ID_BROKER_baseUrl",
```

Comment:
I ask because I don't know: is this supposed to be `ID_BROKER_baseUrl` (like URL) or `ID_BROKER_baseUri` (like URI)?

Reply:
The new `_baseUrl` format is correct.